src/MongoDB.Driver.Core/MongoDB.Driver.Core.csproj

```diff
@@ -126,4 +126,8 @@
     <None Include="..\..\THIRD-PARTY-NOTICES" Pack="true" PackagePath="\" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="System.Net.Http" />
+  </ItemGroup>
+
```

Comment:
This is an inevitable requirement for using `HttpClient` ([link](https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclient?view=netframework-4.8)).

Reply:
Should we be using a `PackageReference`, targeting https://www.nuget.org/packages/System.Net.Http/ instead of a `Reference`?  I _feel_ like `Reference` only works if the user already has that dll/package somewhere in their path but I'm not 100% on that...